oxla/benchmark.sh

```diff
@@ -16,25 +16,25 @@ wget --no-verbose --continue 'https://datasets.clickhouse.com/hits_compatible/hi
 echo "Unpack dataset."
 gzip -d hits.csv.gz
 chmod 777 ~ hits.csv
-mkdir data
+mkdir ~/data
```

Comment:
Could you change this so that all artifacts are stored directly in `oxla/` instead of `~/data/`? (asking because it is convenient to run `git reset --hard` in the ClickBench repository for cleanup after a benchmark run)

Related to that, chmod-ing ~ with 777 seems undesirable.

EDIT: Ah, I see ... /data is mapped into the Docker image. Could you nevertheless try to create `data/` as a subdirectory of `ClickBench/oxla/`?

Reply:
In l. 4-7, we can instead simply install docker via the operating system repository:

```sh
sudo apt install docker.io
```